src/fireworks.ts

```diff
@@ -336,6 +338,8 @@ export class Fireworks {
     this._ctx.fillStyle = `rgba(0, 0, 0, ${this.opacity})`
     this._ctx.fillRect(0, 0, this._width, this._height)
     this._ctx.globalCompositeOperation = 'lighter'
+    this._ctx.lineCap = 'round'
+    this._ctx.lineJoin = 'round'
```

Comment:
Before this change, the fireworks looked like confetti 🤣

Reply:
For thick lines this was indeed necessary to avoid the confetti effect. This said, you might want to have this configurable, maybe. Perhaps people want some confetti fireworks...

Reply:
> For thick lines this was indeed necessary to avoid the confetti effect. This said, you might want to have this configurable, maybe. Perhaps people want some confetti fireworks...

What do you think about this? 👉  4cb8fa9

Reply:
Looks good. However, the line joins do not necessarily need to be adjustable, as there are quite few (if any) line joins. I just added both as I never remember what was used what for and tend to have it either all round or all sharp.